src/map/script.c

```diff
@@ -2909,13 +2917,18 @@ struct script_data *get_val(struct script_state* st, struct script_data* data) {
 		} else
```

Comment:
since the command is rewritten heavily, please apply coding style here too. If you do an if / else where one is using brackets, you do it for the other condition as well even if not needed
`} else {